src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -48,13 +51,18 @@ let idPool = 0;
 		<div class="tabbedPanel fullsize" [ngClass]="options.layout === NavigationBarLayout.vertical ? 'vertical' : 'horizontal'">
 			<div *ngIf="!options.showTabsWhenOne ? _tabs.length !== 1 : true" class="composite title">
 				<div class="tabContainer">
-					<div class="tabList" role="tablist" scrollable [horizontalScroll]="AutoScrollbarVisibility" [verticalScroll]="HiddenScrollbarVisibility" [scrollYToX]="true">
+					<div *ngIf="options.layout === NavigationBarLayout.vertical" class="action-container">
+						<span [title]="toggleTabLabelButtonLabel">
+							<div role="button" [attr.aria-expanded]="toggleTabLabelButtonExpanded" [attr.aria-label]="toggleTabLabelButtonLabel" [ngClass]="toggleTabLabelButtonClass" tabindex="0" (click)="updateToggleState()" (keyup)="onKey($event)"></div>
```

Comment:
Is there a reason you're not having this be an actual button? We should try to use elements that actually correspond to what the thing is - the portal does this currently with their button 

![image](https://user-images.githubusercontent.com/28519865/74966753-88191380-53cc-11ea-8265-f7c07da52e89.png)

Reply:
good catch, after working with ADS repro for such a long time, I tend to use div to build everything :)

changed it to button element now.